doc/book/admin/access_control.rst

```diff
@@ -295,83 +890,179 @@ no-password user becomes this function's creator. Finally, another user
+        *   -   'role'
+            -   A :ref:`role <access_control_concepts_roles>`.
+        *   -   'space'
+            -   A :ref:`space <index-box_space>`.
+        *   -   'function'
+            -   A :ref:`function <box_schema-func_create>`.
+        *   -   'sequence'
+            -   A :ref:`sequence <index-box_sequence>`.
+        *   -   'lua_eval'
+            -   Executing arbitrary Lua code.
+        *   -   'lua_call'
+            -   Calling any global user-defined Lua function.
+        *   -   'sql'
+            -   Executing an arbitrary SQL expression.
+
+
+
+.. _access_control_list_privileges:
+
+Privileges
+~~~~~~~~~~
+
+..  container:: table
+
+    ..  list-table::
+        :header-rows: 1
+        :widths: 15 15 15 55
+
+        *   -   Privilege
+            -   Object type
+            -   Granted to roles
+            -   Description
+        *   -   'read'
+            -   All
+            -   Yes
+            -   Allows reading data of the specified object.
+                For example, this privilege can be used to allow a user to select data from the specified space.
+        *   -   'write'
+            -   All
+            -   Yes
+            -   Allows updating data of the specified object.
+                For example, this privilege can be used to allow a user to modify data in the specified space.
+        *   -   'create'
+            -   All
+            -   Yes
+            -   Allows creating objects of the specified type.
+                For example, this privilege can be used to allow a user to create new spaces.
+
+                Note that this privilege requires read and write access to certain system spaces.
+        *   -   'alter'
+            -   All
+            -   Yes
+            -   Allows altering objects of the specified type.
+
+                Note that this privilege requires read and write access to certain system spaces.
+        *   -   'drop'
+            -   All
+            -   Yes
+            -   Allows dropping objects of the specified type.
+
+                Note that this privilege requires read and write access to certain system spaces.
+        *   -   'execute'
+            -   'role', 'universe', 'function', 'lua_eval', 'lua_call', 'sql'
+            -   Yes
+            -   For 'role', allows using the specified role.
+                For other object types, allows calling a function.
+        *   -   'session'
+            -   'universe'
+            -   No
+            -   Allows a user to connect to the instance over IPROTO.
+        *   -   'usage'
+            -   'universe'
+            -   No
+            -   Allows a user to use their privileges on database objects (for example, read, write, and alter spaces).
+
+
+.. _access_control_list_objects_and_privileges:
+
+Object types and privileges
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  container:: table
+
+    ..  list-table::
+        :header-rows: 1
+        :widths: 15 85
+
+        *   -   Object type
```

Comment:
Wouldn't this look better in a three-column table?

Reply:
Hmm.. Thought about this but cell merging is required for better readability. Does Sphinx support this?

Reply:
AFAIR, no: tables in Sphinx/RST are a bit limited.